codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddBuiltinPlugins.java

```diff
@@ -67,6 +67,11 @@
                         .withConventions("@aws-sdk/middleware-expect-continue", "^0.1.0-preview.5",
                                         "AddExpectContinue", HAS_MIDDLEWARE)
                         .servicePredicate((m, s) -> s.getId().getName().equals("AmazonS3"))
+                        .build(),
+                RuntimeClientPlugin.builder()
+                                .withConventions("@aws-sdk/middleware-sdk-glacier", "^0.1.0-preview.7",
```

Comment:
Nit: Can you line up the indentation with the others?

Reply:
fixed in cd4c2b99686d722ba015aae49e75a7b5e72e7067